### ISSUE TITLE
NullPointerException in OnOffFilter starting RODA

### ISSUE THREAD

**Issue by rui-castro** [keeps/roda#743]
## Description

Every time I start RODA in my IDE I see this exception:

```
Starting Jetty on port 8888
   [WARN] /api/v1/theme
java.lang.NullPointerException
	at org.roda.wui.filter.OnOffFilter.doFilter(OnOffFilter.java:77)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1474)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:499)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:137)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:557)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:231)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1086)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:428)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1020)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.handler.RequestLogHandler.handle(RequestLogHandler.java:68)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:370)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:489)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:949)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:1011)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:644)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:668)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:52)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:745)
[ERROR] 500 - GET /api/v1/theme?resource_id=cookies/cookieconsent.js (127.0.0.1) 3597 bytes
```

### Analysis

This is happening because method [OnOffFilter.isOn()](https://github.com/keeps/roda/blob/hsilva_dev/roda-ui/roda-wui/src/main/java/org/roda/wui/filter/OnOffFilter.java#L96) is not thread safe. Multiple calls are being done simultaneously and the value of [`this.isOn` is being set](https://github.com/keeps/roda/blob/hsilva_dev/roda-ui/roda-wui/src/main/java/org/roda/wui/filter/OnOffFilter.java#L127) before the inner class is actually instantiated.

**Comment by rui-castro** [comment]
@hsilva-keep @luis100 fixed in PR #744

**Comment by hsilva-keep** [comment]
PR #744 merged & closed.
